.gitlab-ci.yml

```diff
@@ -114,6 +114,7 @@ default:
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME == "master"
     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
+    - if: $CI_COMMIT_MESSAGE  =~ /.*build-for-simnet.*/
```

Comment:
Looks like this line should be removed before the merge too. Or is it exist for any additional non-debug purposes?

Reply:
Agree, I can add a  "revert me" line here like

    `- if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/ # PRs  FIXME: delete me`